Account & Project/1. User guides/12. 計畫錢包管理.md

```diff
@@ -1 +1,65 @@
+---
+title: 計畫錢包管理 | en
+tags: Guide, Service, EN
+GA: UA-155999456-1
+---
+
+
+{%hackmd @docsharedstyle/twccheader-en %}
+{%hackmd @docsharedstyle/default %}
+
+:::warning
+==Need to update:exclamation:==
+<i class="fa fa-bullhorn" aria-hidden="true"></i> If your [<ins>Member Center <i class="fa fa-question-circle fa-question-circle-for-service" aria-hidden="true"></i></ins>](https://man.twcc.ai/@twsdocs/howto-service-access-service-zh) not applicable for the footage or process below, please refer to this <i class="fa fa-sign-out" aria-hidden="true"></i> [<ins>link</ins>](https://man.twcc.ai/@twsdocs/doc-mber-pjct-blng-main-zh/https%3A%2F%2Fman.twcc.ai%2F%40twsdocs%2Fguide-service-signup-zh) for manual instructions.
+:::
+
+
+# Project wallet management
+
+The project host or administrator (tenant admin) can perform wallet management for your project members (tenant users) to limit the amount that the members can use during the execution of the project, which helps the project host to effectively Manage the usage of project members so as not to overuse the plan credit.
+
+**If not specifically set, all members share the credit of the main wallet.**
+
+:::info
+<i class="fa fa-laptop" aria-hidden="true"></i> Example of sub-wallet management: a plan to subscribe for 50,000 dollar (main wallet)
+- Member A allocates 10,000 dollar to the sub-wallet
+- Member B allocates 30,000 dollar to the sub-wallet
+- The sub-wallet has been allocated 40,000 dollar, and the main wallet has only 10,000 dollar left
+:::
+
+## Instructions
+
+**Step 1.**  After sign in to TWCC, please click on the username in the upper right corner and select "**Account & Project**"
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_d68fe214ac0a1e1c01463fe553e8a3be.png)
+
+**Step 2.** After enter [Member Center <i class="fa fa-question-circle fa-question-circle-for-service" aria-hidden="true"></i>](https://man.twcc.ai/@twsdocs/howto-service-access-service-zh) ==Need to update:exclamation:== , select: "**Member Center**"> "**Projects**"> "**My Projects**"
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_0ade6dcf56638576f83f787a18a12097.png)
+
+**Step 3.** Select the project to manage member wallets
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_4de493294052af092cf63c68251c5ba6.png)
+
+**Step 4.** Switch to the "**Wallet Management**" tab and click "**Edit**"
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_a4d633084361037a80f2d6f8572efa4b.png)
+
+**Step 5.** When there are two or more members in the project, a list of shared main wallets will be displayed on the left. You can click the member, and click the color arrow to put the member to the "**Exclusive Sub Wallet**" on the right, and then press "**OK**", the member will have an exclusive sub-wallet.
```

Comment:
[SUG]

1. a list of shared main wallets > a list of members shared the main wallet
2. color arrow > green arrow (I'll modify the Mandarin typo lol)